doc/source/guide/introduction/getting_started.rst

```diff
@@ -2,4 +2,288 @@
+       def adjoint(self):
+           return self  # the adjoint is the same as the operator
+   
+Now lets try with the simple `landweber` solver:
+
+.. code-block:: python
+
+   # Need operator norm for step length (omega)
+   opnorm = odl.power_method_opnorm(conv, niter=10)
+   
+   x = space.zero()
+   odl.solvers.landweber(conv, x, data, niter=100, omega=1/opnorm**2)
+   x.show('landweber solution')
+   
+.. image:: figures/getting_started_landweber.png
+
+This solution is not very good, mostly due to the ill-posedness of the convolution operator. 
+Other solvers, like the `conjugate_gradient` solver give similar results:
+
+.. code-block:: python
+
+   x = space.zero()
+   odl.solvers.conjugate_gradient(conv, x, data, niter=100)
+   x.show('conjugate gradient solution')
+
+.. image:: figures/getting_started_conjugate_gradient.png
+
+A method to remedy this problem is to instead consider a regularized problem. One of the classic regularizers is `Tikhonov regularization <https://en.wikipedia.org/wiki/Tikhonov_regularization>`_ where we instead study the problem
+
+.. math::
+   
+   \min_f ||Af - g||_2 + ||Bf||_2
+
+Where :math:`B` is a regularizer. To solve this with the above solvers, we can find the first order optimality conditions
+
+.. math::
+ 
+   2 A^* (Af - g) + 2 B^* B f =0
+
+Where :math:`A^*` is the adjoint of :math:`A`. This can be rewritten on the form :math:`Ax=b`:
+
+.. math::
+
+   (A^* A + B^* B) f = A^* g
+
+We can try solving this with :math:`B` as the `IdentityOperator` using ODL:
+
+.. code-block:: python
+
+   I = odl.IdentityOperator(space)
+   reg_op = conv.adjoint * conv + 0.1 * I.adjoint * I
+   rhs = conv.adjoint(data)
+   
+   x = space.zero()
+   odl.solvers.conjugate_gradient(reg_op, x, rhs, niter=100)
+   x.show('Tikhonov identity conjugate gradient solution')
+
+.. image:: figures/getting_started_tikhonov_identity_conjugate_gradient.png
+
+Slightly better, but no major upgrade. What about letting :math:`B` be the `Gradient`?
+
+.. code-block:: python
+
+   grad = odl.Gradient(space)
+   reg_op = conv.adjoint * conv + 0.0001 * grad.adjoint * grad
+   rhs = conv.adjoint(data)
+   
+   x = space.zero()
+   odl.solvers.conjugate_gradient(reg_op, x, rhs, niter=100)
+   x.show('Tikhonov gradient conjugate gradient solution')
+
+.. image:: figures/getting_started_tikhonov_gradient_conjugate_gradient.png
+
+Perhaps a bit better, but far from excellent. Lets try more modern methods, like TV regularization. Here we want to solve the problem
+
+.. math::
+
+   \min_{0 \leq f \leq 1} ||Af - g||_2 + ||\nabla f||_1
+
+Since this is a non-differentiable problem we need more advanced solvers to solve this problem. One of the stronger solvers in ODL is the Douglas-Rachford Primal-Dual solver (`douglas_rachford_pd`) which uses :ref:`proximal_operators` to solve the optimization problem. Several examples of this and similar solvers (`forward_backward_pd`, `chambolle_pock_solver`, etc) are available in the ODL examples folder.
```

Comment:
"Several examples of this and similar solvers"
Better: "There are several examples available using this solver as well as similar optimization methods, e.g. [...]"
"Several examples available" could be a hyperlink to the examples folder, to be replaced by a link to corresponding tutorials.